runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -128,6 +127,7 @@
 	 */
 	private final Deque<ClanJoinMessage> clanJoinMessages = new ArrayDeque<>();
 	private Map<String, ClanMemberActivity> activityBuffer = new HashMap<>();
+	private int join_leave_duration;
```

Comment:
It is not necessary store this here, you can just call `config.joinLeaveTimeout()` in `timeoutClanMessages`